practice/flutter/flutter/logic.py

```diff
@@ -0,0 +1,20 @@
+"""flutter Logic."""
```

Comment:
Looks like you might have worked on the Flutter problem in the same branch? Come and find me and we can use git to extract the commits with these files into a separate branch.